.gitignore

```diff
@@ -1,2 +1,4 @@
 node_modules
 yarn.lock
+.idea/
+.vscode/
```

Comment:
These should be in your own global gitignore, not in this repo. They're not project-specific.

https://help.github.com/en/articles/ignoring-files#create-a-global-gitignore

Reply:
You are right @sindresorhus, thanks for fix it